aws/elasticache_test.go

```diff
@@ -0,0 +1,99 @@
+package aws
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elasticache"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/go-commons/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+func createTestElasticacheCluster(t *testing.T, session *session.Session, name string) {
+	svc := elasticache.New(session)
+
+	param := elasticache.CreateCacheClusterInput{
+		CacheClusterId: awsgo.String(name),
+		CacheNodeType:  awsgo.String("cache.t3.micro"),
+		Engine:         awsgo.String("memcached"),
+		NumCacheNodes:  awsgo.Int64(1),
+	}
+
+	_, err := svc.CreateCacheCluster(&param)
+	if err != nil {
+		assert.Failf(t, "Could not create test Elasticache cluster", errors.WithStackTrace(err).Error())
+	}
```

Comment:
NIT: We've been slowly transitioning error assertions to use `require.NoError(t, err)`, which simplifies the code while providing useful error logs when the assertion fails, and also halting the test if there is an error in the step.

Reply:
Thanks! I've switched over to using it